modules/virt-add-disk-to-vm.adoc

```diff
@@ -27,8 +27,8 @@ Use this procedure to add a virtual disk to a {object}.
 .Procedure
 
 . Click *Workloads* -> *Virtualization* from the side menu.
-. Click the *{object-gui}s* tab.
-. Select a {object} to open the *{object-gui} Overview* screen.
+. Click the *{object-gui}* tab.
+. Select a {object} to open the *Virtual Machine Overview* pane.
```

Comment:
Hard coding this instead of using the variable means that in the Template assembly this reads:
"Select a virtual machine template to open the Virtual Machine Overview tab."
I believe selecting a template opens the 'VM Template Details' 
https://cnv-9030-vm-templates--ocpdocs.netlify.app/openshift-enterprise/latest/virt/vm_templates/virt-editing-vm-template.html#virt-vm-add-disk_virt-editing-vm-template

Seems to also be the case in the following modules:
virt-add-nice-to-vm.adoc
virt-editing-vm-web.adoc

Reply:
Resolved.